quickwit/quickwit-serve/src/elastic_search_api/bulk.rs

```diff
@@ -78,23 +79,37 @@ pub fn es_compat_index_bulk_handler(
             elastic_ingest_bulk(Some(index), body, ingest_option, ingest_service)
         })
         .and(extract_format_from_qs())
-        .map(make_json_api_response)
+        .map(make_elastic_api_response)
+}
+
+fn make_elastic_api_response(
```

Comment:
there's a similar function in [rest_handler.rs](https://github.com/quickwit-oss/quickwit/blob/ac983a77a1f061644956f32a7d1ce09f32582d15/quickwit/quickwit-serve/src/elastic_search_api/rest_handler.rs#L477), probably could be moved to a more appropriate place

Reply:
Yes, I would keep one function that takes a generic in argument